packages/labs/react/src/test/create-component_test.tsx

```diff
@@ -311,8 +341,8 @@ suite('createComponent', () => {
     assert.equal(el.getAttribute('id'), wrappedEl.getAttribute('id'));
     assert.equal(el.id, wrappedEl.id);
 
-    // @ts-expect-error
-    await renderReactComponent({id: null});
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    await renderReactComponent({id: null as any});
```

Comment:
I'm not sure that these should explicitly be expecting TS error which implies that we're testing that the typing of the created component should not be allowed to take `null` as the value of these HTML attributes.
My thought is we're only concerned with testing runtime behavior of what happens if `null` is somehow provided.